Traceback (most recent call last):
  File "run_experiments.py", line 151, in <module>
    with open(f'{dir_path}/{experiment}_seed_{seed}.pkl', 'wb') as f:
FileNotFoundError: [Errno 2] No such file or directory: 'experiments/dropOutState/closing_window/T_2_H_50_N_5_B_1/T_2_H_50_N_5_B_1_seed_14.pkl'
